src/lit-html.ts

```diff
@@ -394,17 +395,20 @@ export class NodePart implements SinglePart {
   }
 }
 
-
-export type PartCallback = (instance: TemplateInstance, templatePart: TemplatePart, node: Node) => Part;
-
-export const defaultPartCallback = (instance: TemplateInstance, templatePart: TemplatePart, node: Node): Part => {
-  if (templatePart.type === 'attribute') {
-    return new AttributePart(instance, node as Element, templatePart.name!, templatePart.strings!);
-  } else if (templatePart.type === 'node') {
-    return new NodePart(instance, node, node.nextSibling!);
-  }
-  throw new Error(`Unknown part type ${templatePart.type}`);
-}
+export type PartCallback = (instance: TemplateInstance,
+                            templatePart: TemplatePart, node: Node) => Part;
+
+export const defaultPartCallback =
+    (instance: TemplateInstance, templatePart: TemplatePart, node: Node):
```

Comment:
not sure how this is better lol

Reply:
I'm guessing it was to fix line length? /shrug

Reply:
yep. You take the bad with the consistency